SoftwareEngineering/SoftwareEngineering.md

```diff
@@ -0,0 +1,52 @@
+# SoftwareEngineering 💡 
+<br>
+<p align="center"><img src="https://i.ibb.co/RCpPrPw/software-engineering-introduction2.png" height="120"></p>
+<br>
+Software is more than just a program code. A program is an executable code, which serves some computational purpose. Software is considered to be collection of executable programming code, associated libraries and documentations. Software, when made for a specific requirement is called software product.
+
+Engineering on the other hand, is all about developing products, using well-defined, scientific principles and methods.
+
+## Major Topic Under Software Engineering : 💡
+
+* Software Development Life Cycle
+* Software Project Management
+* Software Requirements
+* Software Testing & Debugging
+* Software Development Models & Architecture
+
+## Links for online resources for above mentioned topics : 💡
+
+* [Models in Software Engineering](https://www.tutorialspoint.com/sdlc/sdlc_waterfall_model.htm)
+* [Software Engineering Requirements from Geeks For Geeks](https://www.geeksforgeeks.org/software-engineering-requirements-engineering-process/)
+* [Principles Of Software Testing](https://www.geeksforgeeks.org/software-engineering-seven-principles-of-software-testing/)
+* [Overview On Software Development Life Cycle](https://www.tutorialspoint.com/sdlc/sdlc_quick_guide.htm)
+
+## Books useful for SE : 💡
+
+* [A Down-To-Earth Guide To SDLC Project Management](https://www.amazon.com/exec/obidos/ASIN/1480038199/httpwwwtuto0a-20)
+* [Head First Software Development: A Learner's Companion to Software Development](https://www.amazon.com/exec/obidos/ASIN/0596527357/httpwwwtuto0a-20)
+* [Software Testing: Testing Across the Entire Software Development Life Cycle](https://www.amazon.com/exec/obidos/ASIN/047179371X/httpwwwtuto0a-20)
+* [Human-Centered Software Engineering](https://www.amazon.com/exec/obidos/ASIN/9048170168/httpwwwtuto0a-20)
+* [Fundamentals of Software Engineering](https://www.amazon.in/Fundamentals-Software-Engineering-Rajib-Mall/dp/9388028023/ref=sr_1_2?adgrpid=61770370787&dchild=1&ext_vrnc=hi&gclid=Cj0KCQjw8fr7BRDSARIsAK0Qqr5T1OSxOKj16qZdYnHhDS3zDfgdNiLdgmXK64k0UrvOf8O4Edj7jJUaAsQrEALw_wcB&hvadid=294118818135&hvdev=c&hvlocphy=1007823&hvnetw=g&hvqmt=b&hvrand=6561455418934369109&hvtargid=aud-899312378062%3Akwd-321157394874&hydadcr=25371_1776657&keywords=software+engineer+book&qid=1602161273&sr=8-2&tag=googinhydr1-21)
+
+* Introductory Course On Software Engineering - [Click Here](https://www.edx.org/course/software-engineering-introduction)
```

Comment:
If you could make this particular book link like the other books that you have linked, it would look symmetric

Reply:
Yeah done